src/subscription_manager/gui/about.py

```diff
@@ -34,6 +34,7 @@
             "Red Hat trademarks are not licensed under GPLv2. No permission is "
             "granted to use or replicate Red Hat trademarks that are incorporated "
             "in this software or its documentation.\n")
+AUTHORS = [line.strip() for line in open("%s/docs/authors.txt" % os.getcwd(), 'r')]
```

Comment:
AUTHORS isn't a constant here so ALLCAPS is a little bit misleading.
(Though we aren't particular consistent about that, namely the odd CFG in a few modules)